db/mongoose.js

```diff
@@ -201,6 +205,26 @@ const getMedians = async (req, res, next) => {
     res.json(theResults);
 };
 
+const twapCleaner = async () => {
+  const response = await fetch(assetURI);
+  const data = await response.json();
+  
+  for (const assets in data) {
+    const assetDetails = data[assets];
+    for (const asset in assetDetails) {
+      const empContract = new web3.eth.Contract(EMPContract.abi, assetDetails[asset].emp.address);
+      const currentContractTime = await empContract.methods.getCurrentTime().call();
+      const expirationTimestamp = await empContract.methods.expirationTimestamp().call();
+      const isExpired = Number(currentContractTime) >= Number(expirationTimestamp);
```

Comment:
instead of these you can replace it with one check, if asset is expired calling the emp contract function `isExpired`, then if true filtering all db entries of the related emp address and deleting them, without checking any timestamps